### PR TITLE
Do/build plugin

### PR DESCRIPTION
**Pull request by haensl**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          && fs.existsSync(absSrc)
+          && fs.statSync(absSrc).isFile()) {
+          try {
+            const svg = $(fs.readFileSync(absSrc, 'utf8'));
+            if (options.attrs) {
+              Object.keys(img[0].attribs).filter((attr) => options.attrs.test(attr))
+                .forEach((attr) => svg.attr(attr, img.attr(attr)));
+            }
+            img.after($(svg));
+            img.remove();
```
Comment (f-masche) [inline index.js]: Use `replaceWith` instead.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "cheerio": "^1.0.0-rc.2",
+    "gulp-util": "^3.0.8",
+    "through2": "^2.0.3"
+  },
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "gulp": "^3.9.1",
+    "gulp-util": "^3.0.8",
```
Comment (f-masche) [inline package.json]: `gulp-util` is already in runtime dependencies.

**Comment by haensl** [comment]
@f-masche Please re-review ;)

File: index.js [hunk 3]
```diff
@@ -13,6 +13,9 @@ const defaults = {
   attrs: /^(?!src).*$/,
   root: __dirname
 };
+const process = (img, options, callback) => {
+  return false;
+};
```
Comment (f-masche) [inline index.js]: What is this?

File: index.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  selectors: [
+    'img[src$=".svg"]',
+    'svg[src$=".svg"]'
+  ],
+  attrs: /^(?!src).*$/,
+  root: __dirname
+};
+const process = (img, options, callback) => {
+  return false;
+};
```
Comment (f-masche) [inline index.js]: Please remove this function

**Pull request merged**